src/modules/SX126x.h

```diff
@@ -751,6 +752,10 @@ class SX126x: public PhysicalLayer {
 
     int16_t setFrequencyRaw(float freq);
 
+    /*!
+      \brief Fixes overly eager PA clamping on SX1262 / SX1268, as described in section 15.2 of the datasheet
+    */
+    int16_t fixPaClamping();
```

Comment:
I don't think this method needs to be public - according to the datasheet, the register should be only set during initialization, right?